scripts/workers/submission_worker.py

```diff
@@ -461,6 +467,17 @@ def add_challenge_callback(ch, method, properties, body):
         traceback.print_exc()
 
 
+def edit_challenge_callback(ch, method, properties, body):
+    try:
+        logger.info("[x] Received edit challenge message %s" % body)
+        body = yaml.safe_load(body)
+        process_edit_challenge_message(body)
```

Comment:
As far as the issue is concerned, I think something is missing here also. The file field which have changes needs to be downloaded again and then imported. I think here the code to download those files is missing. 

I may be wrong here, let me know if something is not clear here.